lib/rubocop/cop/ezcater/rspec_match_ordered_array_not_eq.rb

```diff
@@ -0,0 +1,44 @@
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of `match_ordered_array` matcher instead of using `eq` matcher
+      #
+      # @example
+      #
+      #   # good
+      #   expect(foo).to match_ordered_array([1, 2, 3])
+      #   expect(foo).to match_ordered_array [1, 2, 3]
+      #
+      #   # bad
+      #   expect(foo).to eq([1, 2, 3])
+      #   expect(foo).to eq [1, 2, 3]
+      class RspecMatchOrderedArrayNotEq < Cop
```

Comment:
Thoughts on calling this just `RspecMatchOrderedArray`?

Reply:
sure, i was following a convention we were already using with `Ezcater/RspecDotNotSelfDot`

Reply:
maybe "convention" isnt the proper word here, but I was following the naming of another that I saw.

Reply:
Yeah, that was a weird one where just calling it `RSpecDot` didn't make a lot of sense!